handler.js

```diff
@@ -85,20 +94,34 @@ async function handleJob(taskId, rcl, message) {
     logProcInfo = function (pid) {
         // log process command line
         try {
-            let cmdInfo = { "pid": pid, "name": jm["name"], "command": procfs.processCmdline(pid) };
+            let cmdInfo = {"pid": pid, "name": jm["name"], "command": procfs.processCmdline(pid)};
             logger.info("command:", JSON.stringify(cmdInfo));
         } catch (error) {
             if (error.code === ProcfsError.ERR_NOT_FOUND) {
                 console.error(`process ${pid} does not exist`);
             }
         }
 
+
         // periodically log process IO
         logProcIO = function (pid) {
             try {
                 let ioInfo = procfs.processIo(pid);
                 ioInfo.pid = pid;
                 ioInfo.name = jm["name"];
+                if (process.env.HF_VAR_ENABLE_TRACING === "1") {
```

Comment:
`If (otelLogger) { ... }` should be sufficient. That way we avoid too many places where env is tested. The same occurs in other places where otelLogger is used.